ProjectCoimbra.UWP/Project.Coimbra/Pages/GamePage.xaml.cs

```diff
@@ -214,5 +214,37 @@ private void Timer_Tick(object sender, object e)
                 }
             }
         }
+
+        /// Only pause the playback in SOLO mode
```

Comment:
It would be better to use a .NET XML-doc comment here. I'd also suggest adding the rationale here that we discussed earlier today.

```
<summary>
Only pause the playback in SOLO mode.
</summary>
```